### PR TITLE
Set up SELinux for openafs

### PR DESCRIPTION
**Pull request by CheyenneWills**
Adds an additional SELinux module for additional rules and
file contexts in Centos.

Performs a restorecon right before starting the server or the client.
This is to set the contexts for any directories/files created "manually"
during the ansible install.

### CODE REVIEW

**Comment by meffie** [comment]
Howdy, Seems bosserver is not able to start with these changes.

Feb 04 16:04:52 afs01.example.com bosserver[7685]: bosserver: could not set up directories, code 13

File: roles/openafs_client/tasks/CentOS.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-- name: Disable selinux
-  selinux:
-    state: disabled
+#- name: Disable selinux
```
Comment (meffie) [inline roles/openafs_client/tasks/CentOS.yaml]: Hi, I'd prefer if we did not comment out code, just remove it. Thanks.

**Review by meffie — COMMENTED** [review]
Hi, I was thinking we just needed to set the 'seuser' for the files we created. Do we really need to do all this?

https://docs.ansible.com/ansible/latest/modules/file_module.html

File: roles/openafs_client/tasks/CentOS.yaml (hunk 1 above)
Comment (CheyenneWills) [reply roles/openafs_client/tasks/CentOS.yaml]: Done.  Removed the commented blocks in latest commit

**Comment by CheyenneWills** [comment]
Updated the PR.

Removed the commented out blocks for disabling SELinux

Committed the correct .te file

**Pull request merged**